freqtrade/optimize/hyperopt_interface.py

```diff
@@ -191,6 +191,16 @@ def trailing_space(self) -> List[Dimension]:
             Categorical([True, False], name='trailing_only_offset_is_reached'),
         ]
 
+    def trades_space(self) -> List[Dimension]:
```

Comment:
I think at least the function here (it's the function that users can override) should get a better name, like `max_open_trades_space()`.

it'll make it clearer i'd think - and there's no requirement to have the function a "one word" name.